src/fmu/datamodels/fmu_results/enums.py

```diff
@@ -7,48 +7,243 @@ class Classification(StrEnum):
+    """
+
     property = "property"
+    """A property, like permeability or porosity, belonging to a 3D grid.
+
+    Typically provided as an `xtgeo.GridProperty`.
+    """
+
     pvt = "pvt"
+    """Tabular pressure-volume-temperature data.
+
+    Typically provided as a Pandas `Dataframe` for export.
+
+    ```tip
+    You should not export this manually. Use
+    [`SIM2SUMO`](https://fmu-sumo-sim2sumo.readthedocs.io/en/latest/).
+    ```
+    """
+
     regions = "regions"
+    """Distinct areas within the field that have different characteristics.
+
+    Examples may be volume regions or contact regions.
+
+    Typically provided as an `xtgeo.Polygons`.
+    """
+
     relperm = "relperm"
+    """Tabular data relative permeability data.
+
+    Typically provided as a Pandas `Dataframe` for export.
+
+    ```tip
+    You should not export this manually. Use
+    [`SIM2SUMO`](https://fmu-sumo-sim2sumo.readthedocs.io/en/latest/).
+    ```
+    """
+
     rft = "rft"
+    """Tabular reservoir formation tests data.
+
+    ```tip
+    You should not export this manually. Use
+    [`SIM2SUMO`](https://fmu-sumo-sim2sumo.readthedocs.io/en/latest/).
+    ```
+    """
+
     seismic = "seismic"
+    """Data that is seismic in nature, including seismic cubes.
+
+    Typically provided as an `xtgeo.Cube`, `xtgeo.RegularSurface`, or other.
+
+    ```tip
+    This content type requires additional input in the `content_metadata` field.
+
+    Seismic data handling is still immature. More comprehensive data categorization will
+    come in the future.
+    ```
+    """
+
     simulationtimeseries = "simulationtimeseries"
+    """Time-series data generated by a reservoir simulator like OPM Flow or Eclipse.
+
+    For example, a summary file in parsed into a Pandas `Dataframe` by
+    [res2df](https://equinor.github.io/res2df).
+
+    ```tip
+    You should not export this manually. Use
+    [`SIM2SUMO`](https://fmu-sumo-sim2sumo.readthedocs.io/en/latest/).
+    ```
+    """
+
     subcrop = "subcrop"
+    """A polygon indicating an area where a geological layer subcrops.
+
+    Typically provided as an `xtgeo.Polygons` for export.
+    """
+
     thickness = "thickness"
+    """A thickness map.
+
+    Typically provided as an `xtgeo.RegularSurface` for export.
+    """
+
     time = "time"
+    """Tabular data containing time steps.
+
+    Typically provided as a Pandas `Dataframe`.
+    """
+
     timeseries = "timeseries"
+    # Deprecated. Use "simlationtimeseries" ?
+
     transmissibilities = "transmissibilities"
+    """Tabular data containing transmissibilities (neighbour and
+    non-neigbor-connections).
+
+    Typically provided as a Pandas `Dataframe`.
+
+    ```tip
+    You should not export this manually. Use
+    [`SIM2SUMO`](https://fmu-sumo-sim2sumo.readthedocs.io/en/latest/).
+    ```
+    """
+
     velocity = "velocity"
+    """A seismic velocity map represented as a regular surface.
```

Comment:
A velocity could also be a cube (quite common)

"""A seismic velocity field represented as a regular surface or a cube"""